BUILD.md

```diff
@@ -9,103 +9,136 @@ See also README.md Dependencies section
 
 Checkout the .travis.yml for build bootstrap without root/sudo requirement
 
-Notes:
+# Notes:
  On Ubuntu 14.04 - to get g++ 4.9
+ ```
  sudo add-apt-repository ppa:ubuntu-toolchain-r/test
  sudo apt-get upgrade
+ ```
 If using a vmware image/starting fresh
+```
  sudo vmware-config-tools.pl  # to setup shared folders
-
-# Cmake 3.2 or later
-
-See below for mac (Cmake 3.3/head on a mac is recommended for Xcode support)
-
+```
+# Build Instructions
+__Install Cmake 3.2 or greater.__
+*See directions below for Mac
+(Cmake 3.3/head on a Mac is recommended for Xcode support)*
+```
 wget http://www.cmake.org/files/v3.2/cmake-3.2.3.tar.gz
 tar xvfz cmake-3.2.3.tar.gz
 cd cmake-3.2.3
 ./bootstrap --prefix=/usr --parallel=16 && make -j && sudo make install
-
-# Get folly
+```
+__Get folly source tree__
+```
+mkdir deps/; cd deps
```

Comment:
delete this (the instructions in cmake and here have changed, folly should be a sister directory to wdt instead of inside deps (which cause git issues later)